starters/express-typeorm-postgres/README.md

```diff
@@ -0,0 +1,205 @@
+1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
+2. run `npm run infrastructure:start`
+3. run `npm run dev`
+
+The above steps will make sure your API connects to the database and redis instances that gets started up with docker. When you finish work, run `npm run infrastructure:stop` to stop your database and redis containers.
+
+### Seeding
+
+In the `src/db/run-seeders.ts` file, we provide a script to seed the database with intial values, using TypeOrm. Under the `src/db/seeding` folder, you can find the `TechnologySeeder` class, that seeds values into the database as an example.
+
+In order to seed the database, you need to do the following steps:
+
+1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
+2. run `npm run infrastructure:start`
+3. run `npm run db:seed`
+
+### Reset infrastructure
+
+If you for some reason need to clear the contents of your database and you want to reinitialise it, delete the `misc/pg_data` folder and delete the postgres docker container. After that the next `infrastructure:start` command will start up as it would the first time.
+
+If you would like to clear your redis cache and reinitialise it, delete the `misc/cache_conf` and the `misc/cache_data` folders and delete the cache docker container.
+
+If you would like to clear your redis queue and reinitialise it, delete the `misc/queue_conf` and the `misc/queue_data` folders and delete the queue docker container.
+
+### Production build
+
+The `npm run build` command compiles the typescript code into the `/dist` folder and generates a `package.json` file. To use it in production, for example in a docker container, one would copy the contents of the `/dist` folder, and then run `npm install` to have all the dependencies.
+
+### CORS Cross-Origin Resource Sharing
+
+The [Cross-Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) standard works by adding new HTTP headers that let servers describe which origins are permitted to read that information from a web browser. For Security reasons, browsers restrict cross-origin HTTP requests initiated from scripts. This means that you cannot request data from web application on 'https://domain-a.com' from 'https://domain-b.com/data.json'.
+
+This application accepts CORS from all origins by default. Some web applications may require you to add the HTTP header `'Access-Control-Allow-Origin': '*'` to allow access.
+
+In order to restrict origins urls that can access your api, you need to add a list of comma separated origin urls in the `CORS_ALLOWED_ORIGINS` variable located in your `.env` file. For example `CORS_ALLOWED_ORIGINS="https://start.dev"`. In case you need to access the api in a development environment i.e. a sveltekit application, you can add the local url `http://127.0.0.1` to the `CORS_ALLOWED_ORIGINS` variable as `CORS_ALLOWED_ORIGINS=https://start.dev,http://127.0.0.1`.
```

Comment:
Did you want to use `starter.dev` here instead of `start.dev`?

Reply:
Yeah, nice catch